src/tools/ceph_objectstore_tool.cc

```diff
@@ -2601,6 +2660,16 @@ int main(int argc, char **argv)
     goto out;
   }
 
+  if (object == "apply-layout-settings") {
```

Comment:
nit, this is a little bit weird, could also be `--op apply-layout-settings <pool>`.